### DISCUSSION TITLE
SSH Remote: No Output from `open language server log` with Clangd LSP

### DISCUSSION THREAD

**Discussion by ningfangbin** [zed-industries/zed#21908] [Feature Requests]
I have encountered two issues with my SSH remote setup.  First, there is no output when using the `open language server log` command, which works as expected in my local project.  I am using clangd as LSP.  Is this behavior normal for the current version?

Second, I am unable to use `format selection` on the SSH remote server;  it formats my entire file instead of the selected region, , which works as expected in my local project.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
